x-pack/plugins/screenshotting/server/layouts/preserve_layout.css

```diff
@@ -7,6 +7,11 @@
   display: none !important;
 }
 
+/* some elements needs to be stretched to be shared */
+.stretch-for-sharing {
+  margin: 0px;
```

Comment:
I have not found any other solution here to remove the bottom margin in Lens' workspace, used to space the suggestion panel from the chart one.